lib/helper/import.js

```diff
@@ -0,0 +1,100 @@
+'use babel';
+
+const atom = global.atom;
+const Path = require('path');
+const FileSystem = require('fs-plus');
+const Electron = require('electron');
+const Storage = require('./../helper/storage.js');
+const XMLParser = require('xml-js');
+const server_config = require('./../config/server-schema.json');
+const folder_config = require('./../config/folder-schema.json');
+
+class Import {
+  constructor() {
+    const self = this;
+  }
+
+  import(configuration_view) {
```

Comment:
Remove configuration_view. Better use return type of funtion or promise and fetch result un configurationView.

Reply:
Some explanation ? :) I'm still new in nodejs, so don't have a clue how to manage it correctly ;)

Reply:
Done as _callback function as parameter.

Reply:
A Promise would have been better but a callback function does too. Maybe I will replace it later with a promise. It should work the same way.